README.md

```diff
@@ -111,11 +127,13 @@ use Carbon\Carbon;
 
 use App\User;
 
-class AppAzure extends Azure
+class AzureAuthentication extends Azure
 {
     protected function handlecallback($request, Closure $next, $access_token, $refresh_token)
     {
         $user = Auth::user();
+        if($user === null) // If the application looses user data while user is authenticated with Laravel "catch" (i.e: User is deleted)
+            return response()->redirectToRoute('azure.login'); // Make sure the route name is correct if you changed it
```

Comment:
A couple of things here;
1. I'd suggest redirecting to the built in redirect method https://github.com/rootinc/laravel-azure-middleware/blob/master/src/Azure.php#L122
2. We may also want to flush `session()->get('_rootinc_azure_access_token');` and `session()->get('_rootinc_azure_refresh_token');`
3. Though this is an example and might be getting to verbose?

Reply:
This idea is also echoed in the Custom Redirect section as well.

Reply:
I'm not sure how to do nr. 2. Could you help @danieltjewett?
3. Personally I like examples with a lot of content to help me build from. It helps me avoid mistakes other might have done before me, that I eventually will run into.

Reply:
2 and 3 kind of go together.  If we add the proposed in the example, and somehow we get to a spot where user is null (because they were deleted from the DB), we should also add
```
session()-> forget(['_rootinc_azure_access_token', '_rootinc_azure_refresh_token']);
```
in the condition right before we redirect the user back to the login page.  Otherwise, https://github.com/rootinc/laravel-azure-middleware/blob/master/src/Azure.php#L41 would be ignored, potentially reaching an infinite redirect situation.  Adding all of this could be verbose though as (at least originally) the documentation was showing a simple example of how to update the user field.  I do understand your point though.  I'll let you decide 👍   If you think it is for the best to include it all for the documentation, go ahead and make the updates with forgetting the session keys and we can push it through.